### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sibam-Paul**
Updated the readme as u told   #58 

Please see the changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ https://spring-music-player.vercel.app/
  git clone https://github.com/Satyam1923/Spring.git
 ```
 
-### For backend
+### For someone working on backend
```
Comment (Satyam1923) [inline README.md:34]: understand the follow of instructions.

File: README.md [hunk 2]
```diff
@@ -59,9 +57,22 @@ npm install .
 ```sh
 npm run dev
 ```
-4. You'll have local host link in terminal
+### For someone working on backend+frontend
```
Comment (Satyam1923) [inline README.md:60]: these all changes were need to be done in CONTRIBUTING.md

**Review by Satyam1923 — COMMENTED** [review]
Clarify all these changes.

File: README.md (hunk 1 above)
Comment (Sibam-Paul) [reply README.md:34]: ok

**Pull request closed without merging**